shoop/testing/factories.py

```diff
@@ -411,8 +412,11 @@ def create_product(sku, shop=None, supplier=None):
     product.save()
     if shop:
         sp = ShopProduct.objects.create(product=product, shop=shop)
+        if default_price:
```

Comment:
Come to think of it, you can just simply put this in the `.create()` call. If it's `None`, it's NULL; if it's some other value, it just works.